apps/rush-lib/assets/rush-init/rush.json

```diff
@@ -77,7 +77,25 @@
      * After modifying this field, it's recommended to run "rush update --full" so that the package manager
      * will recalculate all version selections.
      */
-    /*[LINE "HYPOTHETICAL"]*/ "resolutionStrategy": "fast"
+    /*[LINE "HYPOTHETICAL"]*/ "resolutionStrategy": "fast",
+
+    /**
+     * If true, then "rush install" will report an error if manual modifications
+     * were made to the PNPM shrinkwrap file without running "rush update" afterwards.
+     *
+     * This feature protects against accidental inconsistencies that may be introduced
+     * if a person manually edits the PNPM shrinkwrap file ("pnpm-lock.yaml").  When this
+     * feature is enabled, "rush update" will append a hash to the file as a YAML comment,
+     * and then "rush update" will validate the hash.  Note that this does not prohibit
+     * manual modifications, but merely requires the person to invoke "rush update"
+     * afterwards, ensuring that PNPM can report or repair any potential inconsistencies.
+     *
+     * To temporarily disable this validation when invoking "rush install", use the
+     * "--bypassPolicy" command-line parameter.
```

Comment:
Don't forget to test the `--bypassPolicy` parameter. 

In so doing, you'll probably notice that it is spelled `--bypass-policy`. 😁

Reply:
Oooooops :)